allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
This line-height is most suitable only for long lines, so think they should be in pair